frontend/src/pages/FeedPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import '../styles/FeedPage.css';
+
+import { Card, Image, Button, Icon, Divider, FeedExtra } from 'semantic-ui-react'
+import { getProfiles } from "../utils/api";
+
+function FeedPage() {
+
+  const [ profiles, setProfile] = useState([]);
+  useEffect(() => {
+    const demoCall = async () => {
```

Comment:
Maybe rename demoCall to something more specific

Reply:
Would loadProfiles be a better name?

Reply:
> Maybe rename demoCall to something more specific

Would loadProfiles be a better name?

Reply:
yea that would be good!